lms/static/js/spec/instructor_dashboard/student_admin_spec.js

```diff
@@ -129,7 +129,58 @@ define(['jquery', 'coffee/src/instructor_dashboard/student_admin', 'js/common_he
                 expect(studentadmin.$request_response_error_ee.text()).toEqual(full_error_message);
             });
 
-            it('binds delete student state for entrance exam ajax call and the result will be success', function() {
+            it('initiates skip entrance exam when button is clicked', function() {
+                studentadmin.$btn_skip_entrance_exam.click();
+                // expect error to be shown since student identifier is not set
+                expect(studentadmin.$request_response_error_ee.text()).toEqual(gettext("Please enter a student email address or username."));
+
+                var success_message = gettext("Student '{student_id}' is marked to skip entrance exam.");
+                var full_success_message = interpolate_text(success_message, {
+                  student_id: unique_student_identifier
+                });
+
+                // Spy on AJAX requests
+                var requests = AjaxHelpers.requests(this);
+
+                studentadmin.$field_entrance_exam_student_select_grade.val(unique_student_identifier)
+                studentadmin.$btn_skip_entrance_exam.click();
+                // Verify that the client contacts the server to start instructor task
+                var url = dashboard_api_url + '/mark_student_can_skip_entrance_exam';
+                AjaxHelpers.expectRequest(requests, 'POST', url, $.param({
+                    'unique_student_identifier': unique_student_identifier
+                }));
+
+                // Simulate a success response from the server
+                AjaxHelpers.respondWithJson(requests, {
+                    message: full_success_message
+                });
+                expect(alert_msg).toEqual(full_success_message);
+            });
+
+            it('shows an error when skip entrance exam fails', function() {
+                // Spy on AJAX requests
+                var requests = AjaxHelpers.requests(this);
+                studentadmin.$field_entrance_exam_student_select_grade.val(unique_student_identifier)
+                studentadmin.$btn_skip_entrance_exam.click();
+                // Verify that the client contacts the server to start instructor task
+                var url = dashboard_api_url + '/mark_student_can_skip_entrance_exam';
+                debugger;
+                AjaxHelpers.expectRequest(requests, 'POST', url, $.param({
+                    'unique_student_identifier': unique_student_identifier
+                }));
+
+                // Simulate an error response from the server
+                AjaxHelpers.respondWithError(requests, 400,{});
+
+                var error_message = gettext("Error marking student '{student_id}' to skip entrance exam." +
+                    " Make sure student identifier is correct.");
```

Comment:
Suggested revisions:

We couldn't find this student ('{student_id}'). Make sure that the student's username or email address is correct and try again.

OR

An error occurred. Make sure that the student's username or email address is correct and try again.